src/vuestic-theme/vuestic-components/vuestic-collapse/VuesticCollapse.vue

```diff
@@ -11,16 +10,28 @@
       <template v-if="noHeader">
         <slot name="header"/>
       </template>
-      <button
+      <div
         v-else
-        class="vuestic-collapse__header__button btn btn-primary"
+        class="vuestic-collapse__header__button-container"
       >
-        <slot name="header"/>
-      </button>
+        <button
+          class="vuestic-collapse__header__button"
+        >
+          <slot name="header"/>
+        </button>
+        <i
+          v-if="show"
+          class="fa fa-angle-up vuestic-collapse__header__button-arrow"
+        />
+        <i
+          v-else
+          class="fa fa-angle-down vuestic-collapse__header__button-arrow"
+        />
+      </div>
     </div>
-
     <div
       class="vuestic-collapse__body"
+      :class="{'background': addBackground}"
```

Comment:
maybe better to use computed class?
and about naming - maybe 'va-collapse__body--with-background' is better option?